lib/Common/Memory/HeapAllocator.h

```diff
@@ -130,6 +130,34 @@ struct HeapAllocator
     static HeapAllocator Instance;
     static HeapAllocator * GetNoMemProtectInstance();
 
+private:
+    HANDLE PrivateHeap;
```

Comment:
I could do that. According to our coding convention it should also start with `m_`, so I'll change it to `m_privateHeap`